nmmo/core/observation.py

```diff
@@ -100,15 +118,193 @@ def to_gym(self):
     if self.config.ITEM_SYSTEM_ENABLED:
       gym_obs["Inventory"] = np.vstack([
         self.inventory.values, np.zeros((
-          self.config.ITEM_N_OBS - self.inventory.values.shape[0],
+          self.config.INVENTORY_N_OBS - self.inventory.values.shape[0],
           self.inventory.values.shape[1]))
       ])
 
     if self.config.EXCHANGE_SYSTEM_ENABLED:
       gym_obs["Market"] = np.vstack([
         self.market.values, np.zeros((
-          self.config.EXCHANGE_N_OBS - self.market.values.shape[0],
+          self.config.MARKET_N_OBS - self.market.values.shape[0],
           self.market.values.shape[1]))
       ])
 
+    gym_obs["ActionTargets"] = self.generate_action_targets()
+
     return gym_obs
+
+  def generate_action_targets(self):
```

Comment:
i would call this _make_action_targets(). Same for helpers: _make_move_mask()

Reply:
changed "generate" to "make" for all functions